assets/js/WfParameter.js

```diff
@@ -500,7 +520,8 @@ export default class WfParameter extends React.PureComponent {
           </div>
         )
 
-      case 'multicolumn':
+      //TODO: Set all multi-column select modules to have type 'multicolumn' for 'colnames'
+      case 'multicolumn' && id_name === 'colnames':
```

Comment:
I'm surprised this compiles -- I've never heard of this syntax.

Could you please nix the `id_name` test (and with it, that nifty syntax)? As I understand it, we'll never reach this code today (because `multicolumn` render is auto-aborted before we get here), but we want this code in the future. Please correct me if I'm wrong....

So the comment should be: `TODO: [what you wrote]; [2018-11-06] right now this code is never reached, but it will be when we finish cleaning up the `multicolumn` parameter type.